### ISSUE TITLE
Crashing When Enabling Event Returners

### ISSUE THREAD

**Issue by Silvenga** [saltstack/salt#54898]
### Description of Issue

After upgrading to 2019.2.1 from 2019.2.0 the existing configuration would crash the master. Reducing the configuration, it appears that enabling any event returner is the source of the crashes.

```
[ERROR   ] An un-handled exception was caught by salt's global exception handler:
TypeError: object.__new__() takes exactly one argument (the type to instantiate)
Traceback (most recent call last):
  File "/usr/bin/salt-master", line 11, in <module>
    load_entry_point('salt==2019.2.1', 'console_scripts', 'salt-master')()
  File "/usr/lib/python3.7/site-packages/salt/scripts.py", line 108, in salt_master
    master.start()
  File "/usr/lib/python3.7/site-packages/salt/cli/daemons.py", line 215, in start
    self.master.start()
  File "/usr/lib/python3.7/site-packages/salt/master.py", line 710, in start
    self.process_manager.add_process(salt.utils.event.EventReturn, args=(self.opts,))
  File "/usr/lib/python3.7/site-packages/salt/utils/process.py", line 445, in add_process
    process = tgt(*args, **kwargs)
  File "/usr/lib/python3.7/site-packages/salt/utils/event.py", line 1172, in __new__
    instance = super(EventReturn, cls).__new__(cls, *args, **kwargs)
TypeError: object.__new__() takes exactly one argument (the type to instantiate)
```

### Setup

master.conf
```
event_return: rawfile_json
```

### Steps to Reproduce Issue

```
docker run -it --rm -e SALT_MASTER_CONFIG='{ "event_return": "rawfile_json" }' saltstack/salt:2019.2.1
```

### Versions Report

```
Salt Version:
           Salt: 2019.2.1

Dependency Versions:
           cffi: 1.11.5
       cherrypy: unknown
       dateutil: Not Installed
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.10.1
        libgit2: 0.28.2
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.6.2
   mysql-python: Not Installed
      pycparser: 2.19
       pycrypto: 2.6.1
   pycryptodome: 3.9.0
         pygit2: 0.28.2
         Python: 3.7.3 (default, May  3 2019, 11:24:39)
   python-gnupg: Not Installed
         PyYAML: 3.13
          PyZMQ: 18.1.0
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.5.3
            ZMQ: 4.3.2

System Versions:
           dist:
         locale: UTF-8
        machine: x86_64
        release: 4.15.0-54-generic
         system: Linux
        version: Not Installed
```

**Comment by Insolita** [comment]
+1, have same, when try to use event_return: pgjsonb, and trouble solved
`
Salt Version:
           Salt: 2019.2.1
 
Dependency Versions:
           cffi: Not Installed
       cherrypy: 3.5.0
       dateutil: 2.5.3
      docker-py: Not Installed
          gitdb: Not Installed
      gitpython: Not Installed
          ioflo: Not Installed
         Jinja2: 2.10
        libgit2: Not Installed
        libnacl: Not Installed
       M2Crypto: Not Installed
           Mako: Not Installed
   msgpack-pure: Not Installed
 msgpack-python: 0.5.6
   mysql-python: Not Installed
      pycparser: Not Installed
       pycrypto: 2.6.1
   pycryptodome: Not Installed
         pygit2: Not Installed
         Python: 3.5.3 (default, Sep 27 2018, 17:25:39)
   python-gnupg: Not Installed
         PyYAML: 3.12
          PyZMQ: 18.1.0 (same with 16.x)
           RAET: Not Installed
          smmap: Not Installed
        timelib: Not Installed
        Tornado: 4.4.3
            ZMQ: 4.3.2
 
System Versions:
           dist: debian 9.9 
         locale: UTF-8
        machine: x86_64
        release: 4.9.0-9-amd64
         system: Linux
        version: debian 9.9 

`

**Comment by Silvenga** [comment]
This was a regression, fixed in #54731.
